### ISSUE TITLE
automated style checking

### ISSUE THREAD

**Issue by mhowlett** [confluentinc/confluent-kafka-python#1351]
we should look at adding automated style checking using `autopep8` or similar.

**Comment by joaoe** [comment]
>  adding automated style checking using autopep8 or similar.

Style checkers are a nightmare to work with, mostly nagging about whitespace.
Always prefer a formatter, like Black if possible, which fixes all whitespace for us.

**Comment by MSeal** [comment]
Agreed on `black` being a preferred option. I've added it to several libraries in the past without much annoyance to repo maintenance while getting linting aid.
